governance/README.md

```diff
@@ -0,0 +1,42 @@
+# opam Repository Governance
+
+This directory holds documents codifying the governance of the opam package
+repository.
+
+## Policies
+
+See the [./policies/](./policies/) directory.
+
+## Maintenance
+
+### Main active maintainers
+
+You are very likely to interact with those maintainers when publishing your package. Don't
+hesitate to get in touch with them if you have questions about the publishing process,
+the CI results, or something else.
+
+- [@mseri](https://github.com/mseri)
+- [@raphael-proust](https://github.com/raphael-proust)
+- [@shonfeder](https://github.com/shonfeder)
+- [@haochenx](https://github.com/haochenx) (in training)
+- [@arbipher](https://github.com/arbipher) (in training)
+
+To contact the maintainers in a more synchronous way you can join the [#opam-repository channel on the OCaml Discord](https://discord.gg/vDH7bMjjYW)
+
+### Full list of all maintainers
+
+- everyone listed above
+- [@avsm](https://github.com/avsm) (repository administrator)
+- [@samoht](https://github.com/samoht) (repository administrator)
+- [@dinosaure](https://github.com/dinosaure)
+- [@damiendoligez](https://github.com/damiendoligez)
+- [@djs55](https://github.com/djs55)
+- [@hcarty](https://github.com/hcarty)
+- [@yallop](https://github.com/yallop)
+- [@dra27](https://github.com/dra27)
+- [@smorimoto](https://github.com/smorimoto)
+- [@dbuenzli](https://github.com/dbuenzli) (triage)
+- [@axrwl](https://github.com/axrwl) (triage)
+- [@octachron](https://github.com/Octachron)
+- The opam developement team (for safe-keeping)
+- [@kit-ty-kate](https://github.com/kit-ty-kate) (retiring maintainer)
```

Comment:
```suggestion
- [@kit-ty-kate](https://github.com/kit-ty-kate) (former maintainer)
```

Reply:
@kit-ty-kate would you prefer to be removed from here to ensure you won't be pinged, or listed as a former maintainer for posterity?

Reply:
mmh, i'm fine, i prefer to be listed. The original reasoning behind the governance document in the wiki was to list everyone who has any kind of special access to the repository for transparency sake.

Reply:
Got it, thanks! I made this change.